kdenlive/umbrel-app.yml

```diff
@@ -0,0 +1,33 @@
+manifestVersion: 1
+id: kdenlive
+category: Media
+name: KdenLive
+version: "421.12.3"
+tagline: Free and open source video editor.
+description: >-
```

Comment:
Due to the use of `>-`, a new line is achieved using 2 new lines. It looks like a few of the sections are missing newlines (spacing between each).

Reply:
done changes for this